src/main/resources/bucketeer.yaml

```diff
@@ -325,3 +325,9 @@ paths:
          schema:
           type: string
           description: The local image path, MUST be a valid path
+       - in: query
+         name: callback-url
+         required: false
+         schema:
+          type: string
+          description: A callback URL for batch job status updates
```

Comment:
I'm not sure I understand what you're proposing to link it to. Do you mean to the Javadocs for the operation ID or ?

Reply:
I meant "link" in the abstract sense of a "connection" -- just that that string should appear somewhere here so that the connection is more clear to a developer

Reply:
(I know I'm not being super consistent with my terminology)